docs/OV_Runtime_UG/model_representation.md

```diff
@@ -0,0 +1,91 @@
+# Model Representation in OpenVINO™ Runtime {#openvino_docs_OV_Runtime_UG_Model_Representation}
+
+In OpenVINO™ Runtime a model is represented by the `ov::Model` class.
+
+## Model Representation
+
+The `ov::Model` object stores shared pointers to `ov::op::v0::Parameter`, `ov::op::v0::Result` and `ov::op::Sink` operations that are inputs, outputs and sinks of the graph.
+Sinks of the graph have no consumers and are not included in the results vector. All other operations hold each other via shared pointers: child operation holds its parent (hard link). If an operation has no consumers and it's not the `Result` or `Sink` operation
+(shared pointer counter is zero), then it will be destructed and won't be accessible anymore. 
+
+Each operation in `ov::Model` has the `std::shared_ptr<ov::Node>` type.
+
+For details on how to build a model in OpenVINO™ Runtime, see the [Build Model from Source](@ref build_model) section.
+
+## Operations
+
+The `ov::Op` class represents any abstract operation in the model representation. Use this class to create [custom operations](../OV_Runtime_UG/Extensibility_DG/AddingNGraphOps.md).
+
+## Operation Sets
+
+Operation set (opset) is a collection of operations that can be used to construct a model. The `ov::OpSet` class  provides a functionality to work with operation sets.
+For each operation set, OpenVINO™ Runtime provides a separate namespace, for example  `opset8`.
+Operation sets allow to differentiate releases of OpenVINO, each release introduces new operations and add these operations to new operation set, also operation sets help to introduce new version of operations which change behavior of old operations. The usage of operation sets allows to avoid changes in the end application if new operations were introduced.
```

Comment:
```suggestion
Each OpenVINO™ Release release introduces new operations and add these operations to a new operation set. New operation sets help to introduce a new version of operations that change behavior of previous operations. Using operation sets allows you to avoid changes in your application if new operations have been introduced.
```